### PR TITLE
added check to remove Buffer type that comes from an old sql server type

### PR DESCRIPTION
**Pull request by aloisbarreras**

### CODE REVIEW

File: lib/MSSQLReadStream.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { logger } from 'aries-data';
 import { Readable } from 'stream';
 import createConnection from './util/createConnection';
+import omitBy from 'lodash.omitBy';
```
Comment (schnie) [inline lib/MSSQLReadStream.js]: `omitBy` should be all lowered.

File: lib/MSSQLReadStream.js [hunk 2]
```diff
@@ -87,4 +89,16 @@ export default class MSSQLReadStream extends Readable {
     async _page() {
         throw new Error('Not Implemented');
     }
-};
+
+    // sql server has a strange data type (either 'timestamp' or 'rowversino') that will comes back looking like this:
```
Comment (schnie) [inline lib/MSSQLReadStream.js]: @aloisbarreras can you make these comments more like jsdoc?  (Just make them look like the other method comments in here)

File: lib/MSSQLReadStream.js (hunk 1 above)
Comment (aloisbarreras) [inline lib/MSSQLReadStream.js]: weird, I swear the test passed. I don't know why it worked

File: lib/MSSQLReadStream.js [hunk 3]
```diff
@@ -87,4 +89,16 @@ export default class MSSQLReadStream extends Readable {
...
+    // we filter these out here
+    removeBufferType(obj) {
+        return omitBy(obj, (value, key) => { 
+            const isObj = isObject(value);
+            const isBuff = value.type === 'Buffer';
+            return isObj && isBuff;
+        });
+    }
+}
+
```
Comment (schnie) [inline lib/MSSQLReadStream.js]: @aloisbarreras Kill the extra newline

File: test/test.js [hunk 4]
```diff
@@ -26,9 +28,17 @@ test('activity proper configuration', t => {
...
 test('index stream', testStream(indexStream));
+
+test('removes buffer type', t => {
+    const activity = new MSSQLReadStream();
+    const obj = { "timestamp_column": { "type":"Buffer", "data":[0,0,0,0,39,195,37,63] }, 'other_column': 'other value' };
+    const newObj = activity.removeBufferType(obj);
+    t.deepEqual(newObj, { 'other_column': 'other value' });
+    t.end();
+});
+
```
Comment (schnie) [inline test/test.js]: @aloisbarreras same here.

File: lib/MSSQLReadStream.js (hunk 3 above)
Comment (aloisbarreras) [inline lib/MSSQLReadStream.js]: didn't we agree that the newline is good

File: lib/MSSQLReadStream.js [hunk 5]
```diff
@@ -87,4 +89,16 @@ export default class MSSQLReadStream extends Readable {
     async _page() {
         throw new Error('Not Implemented');
     }
-};
+
+    // sql server has a strange data type (either 'timestamp' or 'rowversino') that will comes back looking like this:
+    // { "timestamp_column": { "type":"Buffer", "data":[0,0,0,0,39,195,37,63] } }
+    // we filter these out here
+    removeBufferType(obj) {
+        return omitBy(obj, (value, key) => {
```
Comment (schnie) [inline lib/MSSQLReadStream.js]: @aloisbarreras kill the trailing space here.

**Comment by schnie** [comment]
@aloisbarreras Looks good, just some formatting things.

**Pull request merged**